Document/0x06d-Testing-Data-Storage.md

```diff
@@ -819,6 +819,8 @@ libdyld.dylib                     0x185c81000  20480 (20.0 KiB)     /usr/lib/sys
 
 ##### Fridump (No Jailbreak needed)
 
+To use Fridump you need to have either a jailbroken/rooted device with Frida-server installed. The only other way this can be achieved is by building the original application with the Frida library attached (instructions on Frida’s site) [here](https://www.frida.re/docs/ios/)
```

Comment:
> To use Fridump you need to have either a jailbroken/rooted device with Frida-server installed.

If you use "either" in your sentence, you have to use two options in the same sentence: "You need to have either a jailbroken device, or build the application the library attached."

> The only other way this can be achieved is by building the original application with the Frida library attached

Put a period on the end of the sentence.

> [here](https://www.frida.re/docs/ios/)

Please use a decriptive text for the link. For example: Instructions can be found on the [Frida website](https://www.frida.re/docs/ios/).

Reply:
Fixed!

Reply:
Fixed!